edge-modules/TestAnalyzer/Controllers/DirectMethodStatusController.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace TestAnalyzer.Controllers
+{
+    using System.Net;
+    using System.Threading.Tasks;
+    using Microsoft.AspNetCore.Mvc;
+
+    [Route("api/[controller]")]
+    [ApiController]
+    public class DirectMethodStatusController : Controller
+    {
+        // POST api/directmethodstatus
+        [HttpPost]
+        public async Task<StatusCodeResult> PostAsync(CloudOperationStatus methodCallStatus)
+        {
+            await ReportingCacheWithStorage.Instance.AddDirectMethodAsync(methodCallStatus);
+            return this.StatusCode((int)HttpStatusCode.OK);
```

Comment:
Refer to this post, https://medium.com/@santhoshkumarkrishna/http-get-rest-api-no-content-404-vs-204-vs-200-6dd869e3af1d.  I will update it to 204 instead.

Reply:
updated